bin/GameIndex.dbf

```diff
@@ -15841,7 +15872,7 @@ Serial = SLES-54308
 Name   = Phantasy Star Universe
 Region = PAL-M3
 Compat = 5
-eeRoundMode = 0
+eeRoundMode = 0 // Fix for ennemies not moving
```

Comment:
`Fixes enemies not moving.`

@Ryudo300 mentioned an issue with Chapter 7's boss that was fixed by setting this to positive. Are you sure this is the best setting?

Reply:
Rounding to positive or nearest does the same thing (solve the entire chapter 7 (Boss include)).

Reply:
Yes, but it mentions an issue with the boss remaining.